nixos/modules/programs/clickshare.nix

```diff
@@ -0,0 +1,16 @@
+{ config, lib, pkgs, ... }:
+
+{
+
+  options.programs.clickshare-csc1.enable =
+    lib.options.mkEnableOption "Barco ClickShare CSC-1 driver/client";
```

Comment:
Probably we should add a line that users need to be added to the `clickshare` group to be able to use it.

Reply:
Done (I hope the interpunctation of `mkEnableOption` adds up correctly in the documentation).